File: leet_code/27/27.py
def removeElement(nums,val):
    k = 0
    for i in range(len(nums)):
        if nums[i]!=val:
            nums[k] = nums[i]
            k+=1
    return k

def main():
    nums = [3,2,2,3]
    val = 3
    k = removeElement(nums,val)
    print("K = ",k)
    print("Modified nums: ",nums[:k])

if __name__ =="__main__":
    main()

File: leet_code/27/27.js
function removeElement(nums, val) {
  let k = 0;
  for (let i = 0; i < nums.length; i++) {
    if (nums[i] !== val) {
      nums[k] = nums[i];
      k++;
    }
  }
  return k;
}

function main() {
  const nums = [3, 2, 2, 3];
  const val = 3;
  const k = removeElement(nums, val);
  console.log("k : ", k);
  console.log("Modified nums: ", nums.slice(0, k));
}

main();
